packages/yew/src/html/conversion.rs

```diff
@@ -99,6 +120,16 @@ impl_into_prop!(|value: &'static str| -> AttrValue { AttrValue::Static(value) })
 impl_into_prop!(|value: String| -> AttrValue { AttrValue::Rc(Rc::from(value)) });
 impl_into_prop!(|value: Rc<str>| -> AttrValue { AttrValue::Rc(value) });
 
+const TRUE: AttrValue = AttrValue::Static("true");
+const FALSE: AttrValue = AttrValue::Static("false");
+impl_into_prop!(|value: bool| -> AttrValue {
+    if value {
+        TRUE
+    } else {
+        FALSE
+    }
+});
```

Comment:
It was avoid allocating strings for booleans. Do you know of a better approach?

Reply:
I'm really not entirely sure about this but I think that's the kind of stuff the compiler will optimize anyway. `"true"` and `"false"` are `&'static str` after all.

Reply:
Compiler can optimize literals but compiler won't let you pass a bool as an AttrValue without this. I think I should have a manual implementation of the trait instead and use `Option<bool>` instead of `bool`

Reply:
My assumption is that this is meant to be used for HTML boolean attributes. If that's the case however, these values aren't standard compliant. The standard recognizes neither "true" nor "false":

> A number of attributes in HTML5 are boolean attributes. The presence of a boolean attribute on an element represents the true value, and the absence of the attribute represents the false value. 
>
> If the attribute is present, its value must either be the empty string or a value that is a case-insensitive match for the attribute's canonical name, with no leading or trailing whitespace. 


Is there a different reason for this implementation I'm missing?